_includes/tutorials/finding-distinct/ksql/code/docker-compose.yml

```diff
@@ -64,16 +66,16 @@ services:
       KSQL_LISTENERS: "http://0.0.0.0:8088"
       KSQL_CACHE_MAX_BYTES_BUFFERING: 0
       KSQL_KSQL_SCHEMA_REGISTRY_URL: "http://schema-registry:8081"
-      KSQL_PRODUCER_INTERCEPTOR_CLASSES: "io.confluent.monitoring.clients.interceptor.MonitoringProducerInterceptor"
-      KSQL_CONSUMER_INTERCEPTOR_CLASSES: "io.confluent.monitoring.clients.interceptor.MonitoringConsumerInterceptor"
 
   ksql-cli:
-    image: confluentinc/cp-ksql-cli:5.3.2
+    image: confluentinc/ksqldb-server:0.7.1
     container_name: ksql-cli
     depends_on:
       - broker
       - ksql-server
     entrypoint: /bin/sh
+    environment:
+      KSQL_CONFIG_DIR: "/etc/ksql"
```

Comment:
We might want to update the rest of the service versions in this file to be what the others are, which is 5.4.0 I believe

Reply:
ack